### ISSUE TITLE
Blacklisting breaks Action Listener

### ISSUE THREAD

**Issue by derpoho** [zalmoxisus/redux-devtools-extension#245]
Hey,

something is going wrong here, maybe it's because of redux-storage. The following breaks the Action Display after the First occurring action.

```
const composeEnhancers = composeWithDevTools({
  actionsBlacklist: ['REDUX_STORAGE_SAVE'],
})
```

The Inspector Output stops at the first occurence of this Action Type. When i remove the blacklist, everything works fine.

I checked the console of the Extension, it displays an error after the first REDUX_STORAGE_SAVE and keeps popping up this error for every following action:

```
Error in event handler for (unknown): TypeError: Cannot read property 'timestamp' of undefined
    at o (chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/window.bundle.js:16:31659)
    at chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/window.bundle.js:17:1321
    at t.value (chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/window.bundle.js:17:1139)
    at f._renderValidatedComponentWithoutOwnerOrContext (chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/window.bundle.js:6:31476)
    at f._renderValidatedComponent (chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/window.bundle.js:6:31564)
    at f._updateRenderedComponent (chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/window.bundle.js:6:30918)
    at f._performComponentUpdate (chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/window.bundle.js:6:30719)
    at f.updateComponent (chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/window.bundle.js:6:29997)
    at f.receiveComponent (chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/window.bundle.js:6:29180)
    at Object.receiveComponent (chrome-extension://lmhkpmbekcpmknklioeibfkpmmfibljd/js/window.bundle.js:5:9059)
```

When the blacklist is removed, the error doesn't come up.

Thanks for the help!

Edit: the Same happens when i filter the action directly within the Extension Settings.

**Comment by zalmoxisus** [comment]
@derpoho, thanks for the detailed report! Fixed in `2.9.0.1`, please update the extension from Chrome Store and confirm that.

**Comment by derpoho** [comment]
Nice Job as always @zalmoxisus ! Thanks!
